docs/book/src/tasks/automated-machine-management/healthchecking.md

```diff
@@ -211,7 +211,8 @@ This is useful for dynamically scaling clusters where the number of machines kee
 
 There are scenarios where remediation for a machine may be undesirable (eg. during cluster migration using `clusterctl move`). For such cases, MachineHealthCheck provides 2 mechanisms to skip machines for remediation.
 
-Implicit skipping when the resource is paused (using `cluster.x-k8s.io/paused` annotation):
+Implicit skipping when the resource is paused (using `cluster.x-k8s.io/paused` annotation or `Spec.Paused = true` for Cluster or MachineDeployment):
```

Comment:
Please adjust the whole paragraph to:

```
There are scenarios where remediation for a machine may be undesirable (eg. during cluster migration using clusterctl move). For such cases, MachineHealthCheck provides the following mechanisms to skip remediation.

- Users can skip remediation for a specific machine by setting the cluster.x-k8s.io/skip-remediation annotation on it.
- Paused Machines (Machines with the cluster.x-k8s.io/paused annotation) are not considered for remediation.
- If a specific MHC resource is paused (using cluster.x-k8s.io/paused annotation), it will stop to remediate the corresponding target machines.
- If the Cluster is paused (using the cluster.x-k8s.io/paused annotation or by setting cluster.spec.paused to true), all the MHC resources belonging to the Cluster will be implicitly paused, and thus stop remediating target machines.

Note: the last option (pausing the Cluster) is the one used by clusterctl move.
```

xref: https://github.com/kubernetes-sigs/cluster-api/issues/9026#issuecomment-2139920922

Reply:
Thanks, Done